lib/services/usb-livesync-service.ts

```diff
@@ -47,9 +49,7 @@ export class UsbLiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncSer
 				this.$projectDataService.initialize(this.$projectData.projectDir);
 				let frameworkVersion = this.$projectDataService.getValue(platformData.frameworkPackageName).wait().version;
 				if (semver.lt(frameworkVersion, "1.2.1")) {
-					let shouldUpdate = this.$prompter.confirm(
-						"You need Android Runtime 1.2.1 or later for LiveSync to work properly. Do you want to update your runtime now?"
-					).wait();
+					let shouldUpdate = this.$prompter.confirm("You need Android Runtime 1.2.1 or later for LiveSync to work properly. Do you want to update your runtime now?").wait();
```

Comment:
Imho, the UI belongs to another layer - the commands which ultimately depend on the LiveSyncService. Consider moving out this prompt up the chain of callers.